app/controller/pizza/index.ts

```diff
@@ -0,0 +1,24 @@
+import {Router, Request, Response} from 'express';
+import Pizza from "../../model/pizza";
+
+
+let api = Router();
+
+api.post("/", async(req: Request, res: Response) => {
+    const pizza = new Pizza({
+        name: req.body.name,
+        price: req.body.price,
+    });
+    await pizza.save();
+    res.status(200).json({pizza});
+} );
+
+api.get("/get-pizza", async(req: Request, res: Response) => {
```

Comment:
You dont have to name all your routes, consider just using the urls and API verbs to define your API.
If all routes have a name, it quickly becomes complex

Reply:
Also is this a get all or?

Reply:
yeah, wasnt suppose to name that

Reply:
Yes, supposed to get all Pizza